cookbook/doctrine/registration_form.rst

```diff
@@ -213,10 +215,9 @@ into the database::
     // src/AppBundle/Controller/RegistrationController.php
     namespace AppBundle\Controller;
 
-    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-
     use AppBundle\Form\UserType;
     use AppBundle\Entity\User;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
     use Symfony\Component\HttpFoundation\Request;
     use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
```

Comment:
"Sensio" could go before "Symfony" to keep the alphabetical order.

Same for [line 46](#diff-3efb51c932d0635479c8d3bafb824c7fR46) can be the last.

[Line 250](#diff-3efb51c932d0635479c8d3bafb824c7fR250) "[...] like send**ing** [...] ?
[Line 414](#diff-3efb51c932d0635479c8d3bafb824c7fR414) shouldn't it be "[...] Symfony knows __how__ to [...]"